srcpkgs/tailspin/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'tailspin'
+pkgname=tailspin
+version=5.4.2
+revision=1
+build_style=cargo
+#hostmakedepends=""
+#makedepends=""
+#depends=""
+short_desc="Log file highlighter"
+maintainer="zenobit <zenobit@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/bensadeh/tailspin"
+#changelog=""
```

Comment:
https://raw.githubusercontent.com/bensadeh/tailspin/refs/heads/main/CHANGELOG.md and remove the commented out variables